public void foo() throws ReallyBadException {
  laundry.doLaundry();
}
